### DISCUSSION TITLE
Change Support link and Add PAYPAL pls

### DISCUSSION THREAD

**Discussion by RubberArchind** [ZFC-Digital/puppeteer-real-browser#220] [Ideas]
Hi so i notice that you change ZFC to an organization and thats make the support link unusable so you must update it, and if you have Paypal it would be great if you put it there also because github sponsor not support paypal anymore, i will be happy to send you a few bucks hehe.

**Comment by mdervisaygan** [comment]
Thank you very much for both helping people with problems and supporting them financially.
I will update this soon, thank you.
